src/svgs/emojis/index.js

```diff
@@ -1,42 +1,50 @@
+export {
+  default as ConfusedSelectedEmoji,
+} from './confused-selected-emoji.svg';
 export { default as ElatedEmoji } from './elated-emoji.svg';
-export { default as ElatedFilledEmoji } from './elated-filled-emoji.svg';
+export { default as ElatedSelectedEmoji } from './elated-selected-emoji.svg';
 export { default as FuriousEmoji } from './furious-emoji.svg';
-export { default as FuriousFilledEmoji } from './furious-filled-emoji.svg';
+export { default as FuriousSelectedEmoji } from './furious-selected-emoji.svg';
 export { default as HeartEmoji } from './heart-emoji.svg';
-export { default as HeartFilledEmoji } from './heart-filled-emoji.svg';
+export { default as HeartSelectedEmoji } from './heart-selected-emoji.svg';
 export { default as HugEmoji } from './hug-emoji.svg';
-export { default as HugFilledEmoji } from './hug-filled-emoji.svg';
+export { default as HugSelectedEmoji } from './hug-selected-emoji.svg';
 export { default as MildAcneEmoji } from './mild-acne-emoji.svg';
-export { default as MildAcneFilledEmoji } from './mild-acne-filled-emoji.svg';
+export {
+  default as MildAcneSelectedEmoji,
+} from './mild-acne-selected-emoji.svg';
 export { default as NoticeableAcneEmoji } from './noticeable-acne-emoji.svg';
 export {
-  default as NoticeableAcneFilledEmoji,
-} from './noticeable-acne-filled-emoji.svg';
+  default as NoticeableAcneSelectedEmoji,
+} from './noticeable-acne-selected-emoji.svg';
 export { default as OverwhelmedEmoji } from './overwhelmed-emoji.svg';
 export {
-  default as OverwhelmedFilledEmoji,
-} from './overwhelmed-filled-emoji.svg';
+  default as OverwhelmedSelectedEmoji,
+} from './overwhelmed-selected-emoji.svg';
 export { default as PreparedEmoji } from './prepared-emoji.svg';
-export { default as PreparedFilledEmoji } from './prepared-filled-emoji.svg';
+export {
+  default as PreparedSelectedEmoji,
+} from './prepared-selected-emoji.svg';
 export { default as SatisfiedEmoji } from './satisfied-emoji.svg';
-export { default as SatisfiedFilledEmoji } from './satisfied-filled-emoji.svg';
+export {
+  default as SatisfiedSelectedEmoji,
+} from './satisfied-selected-emoji.svg';
 export { default as SevereAcneEmoji } from './severe-acne-emoji.svg';
 export {
-  default as SevereAcneFilledEmoji,
-} from './severe-acne-filled-emoji.svg';
+  default as SevereAcneSelectedEmoji,
+} from './severe-acne-selected-emoji.svg';
 export { default as SmileyEmoji } from './smiley-emoji.svg';
-export { default as SmileyFilledEmoji } from './smiley-filled-emoji.svg';
+export { default as SmileySelectedEmoji } from './smiley-selected-emoji.svg';
 export { default as SparkleEmoji } from './sparkle-emoji.svg';
-export { default as SparkleFilledEmoji } from './sparkle-filled-emoji.svg';
+export { default as SparkleSelectedEmoji } from './sparkle-selected-emoji.svg';
 export { default as SupportedEmoji } from './supported-emoji.svg';
 export { default as SupportedFilledEmoji } from './supported-filled-emoji.svg';
```

Comment:
This one is being removed after the other repos are updated. It's equivalent is `SmileySelectedEmoji`